include/deal.II/fe/fe_q_base.h

```diff
@@ -324,6 +324,12 @@ class FE_Q_Base : public FE_Poly<POLY,dim,spacedim>
    * Mutex for protecting initialization of restriction and embedding matrix.
    */
   mutable Threads::Mutex mutex;
+
+  /*
+   * For the FE_Q_Bubbles class the highest polynomial degree is
+   * greater than the degree of FE_Q by one.
+   */
```

Comment:
No, I don't think so. For FE_Q_Bubbles(p) degree should be p+1 and it is.
The maximum polynomial degree in one direction is given by the bubble enrichments, since these can be written by (x_j^p-x_j^{p+1}) \prod_{i\not=j}x_i(1-x_i).
Maybe the comment is simply confusing. I try to come up with something better.